README.md

```diff
@@ -99,6 +99,30 @@ NAME      AGE
 example   15s
 ```
 
+After a couple of minutes, all cluster members should become ready.
+
+```
+$ kubectl get zookeepercluster
```

Comment:
Could u replace `zookeepercluster` with `zk`

Reply:
I have replaced it.